jvm/src/main/scala/async/VThreadSupport.scala

```diff
@@ -2,19 +2,19 @@ package gears.async
 
 import scala.annotation.unchecked.uncheckedVariance
 import java.util.concurrent.locks.ReentrantLock
-import java.time.Duration
+import scala.concurrent.duration.FiniteDuration
 
-given VThreadScheduler.type = VThreadScheduler
-given VThreadSupport.type = VThreadSupport
+// given VThreadScheduler.type = VThreadScheduler
+// given VThreadSupport.type = VThreadSupport
 
 object VThreadScheduler extends Scheduler:
   override def execute(body: Runnable): Unit = Thread.startVirtualThread(body)
 
-  override def schedule(delay: Duration, body: Runnable): Cancellable =
+  override def schedule(delay: FiniteDuration, body: Runnable): Cancellable =
     val th = Thread.startVirtualThread: () =>
-      Thread.sleep(delay)
-      body.run()
-    () => th.interrupt() // TODO this may interrupt the body after sleeping
```

Comment:
Is there a subject-related reason for removing this note, i.e., did you fix it?

Reply:
Yeah, I basically just end the sleeping thread and put the `body` into the scheduler; that way the body is run in another v-thread and not get interrupted.